docs/image_mirroring.md

```diff
@@ -0,0 +1,39 @@
+# Mirroring images for disconnected environments
+
+You can deploy a cluster with the SiteConfig operator by using the Image Based Install Operator as your underlying operator. If you deploy your clusters with the Image Based Install Operator in a disconnected environment, you must supply your mirror images as extra manifests in the `ClusterInstance` custom resource.
+Complete the following steps:
+
+Create a YAML file for your `ImageDigestMirrorSet` object that contains your mirror registry locations.
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+  name: example-idms
+spec:
+  imageDigestMirrors:
+  - mirrors:
+    - mirror.registry.example.com/image-repo/image
+    source: registry.example.com/image-repo/image
+```
```

Comment:
I think the IDMS is supposed to be embedded inside a ConfigMap as this is what the SiteConfig Operator expects - i.e. a reference to "extra-manifest" ConfigMaps. @trewest can you confirm?

Reply:
Yeah that's right. It would look something like this:
```
---
kind: ConfigMap
apiVersion: v1
metadata:
  name: "extra-manifests-configmap"
  namespace: "example-namespace"
data:
  99-example-idms.yaml: |
    apiVersion: config.openshift.io/v1
    kind: ImageDigestMirrorSet
    metadata:
      name: example-idms
    spec:
      imageDigestMirrors:
      - mirrors:
        - mirror.registry.example.com/image-repo/image
        source: registry.example.com/image-repo/image
```

Reply:
In this case, do we apply the configmap or the resource in the step?
`oc apply -f extra-manifests-configmap.yaml` or `oc apply -f example-idms.yaml`

Reply:
It would be the configmap

Reply:
Thanks for the clarification, I updated the process accordingly. I renamed the configmap to `idms-configmap` to be more fitting for the procedure. PTAL